### ISSUE TITLE
Add new features (like search feature)

### ISSUE THREAD

**Issue by ShraxO1** [ShraxO1/Amazon_Clone_ReactJS#1]

**Comment by irfanukani** [comment]
Hey, For the search, we need to get data from some "API" I guess. Currently, you have statically typed the products in Home.js

**Comment by ShraxO1** [comment]
Yes, You will have to make the changes, I can assign this to you if you want.
